Source/App/EbAppInputy4m.c

```diff
@@ -43,7 +43,10 @@ int32_t read_y4m_header(EbConfig_t *cfg) {
 
     /* get first line after YUV4MPEG2 */
     fresult = fgets(buffer, sizeof(buffer), ptr_in);
-    assert(fresult != NULL);
+    if (fresult== NULL)
```

Comment:
style.md:
Braces go on the same line for single-line statements, but on a new line for multi-line statements: